addons/freifunk-berlin-tunneldigger/files/tunneldigger.init

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh /etc/rc.common
+
+. $IPKG_INSTROOT/lib/functions/network.sh
+
+# Comment out the START declaration to prevent the init file from
+# actually starting tunneldigger at startup.  Instead rely on the
+# hotplug scripts
+#START=90
+
+PIDPATH=/var/run
+tunnel_id=0
+
+tunnel_is_up() {
```

Comment:
the indentication is broken on the following lines

Reply:
fixed with commit edfc1fa